tests/infra_test.go

```diff
@@ -693,12 +696,21 @@ var _ = Describe("[Serial]Infrastructure", func() {
 			}
 
 			errorCount := 0
+			unknownErrorTypeFound := false
 			for ix := 0; ix < concurrency; ix++ {
 				err := <-errors
 				if err != nil {
-					errorCount += 1
+					var e *neturl.Error
+					if errs.As(err, &e) {
```

Comment:
`e` should be populated with the actual error, so why are we checking what it contains? `e.Err`
If you know the embedded error, you can check for it and not the wrapper.

Reply:
this is what other places + the link you sent does, something like a safe cast, can you explain what you suggest?
i will try to play with it, but best if you want to explain

Reply:
I think I was specific here. You look for `neturl.Error` but in practice you check the error it wraps. So I suggest you look for the wrapped error if possible instead of the wrapper.